### ISSUE TITLE
Unused loop

### ISSUE THREAD

**Issue by CVxTz** [ParikhKadam/bidaf-keras#3]
https://github.com/ParikhKadam/bidaf-keras/blob/23bfc46f68e6b7c9f56be0ef3a5d15d587e065f4/bidaf/models/bidaf.py#L19

Hello,

Thanks a lot for your work.
I am very interested in learning from your implementation and I don't get the use of this loop since the passage_layer layer will be only applied once to the unchanged input no matter how many loop iteration there are. Is this the intended behavior ? 

Best,

**Comment by ParikhKadam** [comment]
hey @CVxTz Thank you for informing. No that's absolutely not an intended behavior. When I wrote the code for this model, I was totally new to Deep Learning and this is my first project in the field. I had to refer several articles, codes and open-source modules for getting started. And after that, I never looked at the model's architecture except that I got some errors/bugs. Also, the model's performance is good so it never gave me a thought of looking at the code.

I will be removing this loop in next commit. Also, once I thought of removing highway layers as they don't make much difference but I never tried removing it. I look into this too..

Thank you..
